lib/action_controller/serialization.rb

```diff
@@ -53,11 +45,22 @@ def use_adapter?
       end
     end
 
+    # Tries to rescue the exception by looking up and calling a registered handler.
+    #
+    # Possibly Deprecated
+    # TODO: Either Decorate 'exception' and define #handle_error where it is serialized
+    # For example:
+    #   class ExceptionModel
+    #     include ActiveModel::Serialization
+    #     def initialize(exception)
+    #     # etc
+    #   end
+    #   def handle_error(exception)
+    #     exception_model = ActiveModel::Serializer.build_exception_model({ errors: ['Internal Server Error'] })
+    #     render json: exception_model, status: :internal_server_error
+    #   end
+    # OR remove method as it doesn't do anything right now.
     def rescue_with_handler(exception)
-      @_serializer = nil
-      @_serializer_opts = nil
-      @_adapter_opts = nil
-
       super(exception)
```

Comment:
I've never crossed this method myself, so I lack context. Are you absolutely sure it's not used anywhere? If it's not, I'd advise removing it and then adding a solution in another PR.